src/Controller/PageAdminController.php

```diff
@@ -294,6 +294,7 @@ public function composeContainerShowAction(Request $request): Response
             }
             $container = $template->getContainer($blockCode);
 
+            //TODO is the problem, in 3.x block services have items, but in 4.x it's empty.
```

Comment:
Hey @VincentLanglet in this action I basically just need the getObject, from [AdminInterface](https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Admin/AdminInterface.php#L221), could we create an interface with this? I guess inject the whole AdminInterface to use only one method is a lot, WDYT?

Reply:
I don't see a reason to create another interface. There is no issue using AdminInterface

Reply:
Ok it's not a problem to use the AdminInterface, it gonna work, but don't you think the AdminInterface is quite huge 👀

Reply:
Yes it is huge, but splitting it is a gigantic work, let's keep things simple for now 🙏